pkg/supportbundle/staticspecs/embed.go

```diff
@@ -0,0 +1,58 @@
+package staticspecs
+
+import (
+	_ "embed"
+
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"github.com/replicatedhq/troubleshoot/pkg/supportbundle"
+)
+
+//go:embed clusterspec.yaml
+var clusterspec []byte
+
+//go:embed kurlspec.yaml
+var kurlspec []byte
+
+//go:embed vendorspec.yaml
+var vendorspec []byte
+
+//go:embed defaultspec.yaml
+var defaultspec []byte
+
+var spec *troubleshootv1beta2.SupportBundle
```

Comment:
This variable is unnecessary

Once removed, your getters code will look so

```go
func GetVendorSpec() troubleshootv1beta2.SupportBundle {
	spec, err := supportbundle.ParseSupportBundleFromDoc(vendorspec)
	if err != nil {
		panic(err)
	}
	return *spec
}
```

Reply:
Thank you @banjoh, I have remove unnecessary code and return error back to spec.go to handle.